packages/core/src/services/themes.js

```diff
@@ -8,9 +8,10 @@ function isThemesSupported() {
   }
 }
 
-export function getCSSVar(varName) {
+export function getCSSVar(varName, fallback) {
+  const fb = fallback ? `, ${fallback}` : "";
   if (isThemesSupported()) {
-    return `var(--${varName})`;
+    return `var(--${varName}${fb})`;
```

Comment:
Why is it concatenated?

Reply:
to have a fallback value
`fb` decides on whether of not to add comma, as if I would add comma without a value after it it is not a valid css var